### PR TITLE
apiserver/facades/agent/metricsender: Adding model names to metrics.

### PR DESCRIPTION
**Pull request by alesstimec**
## Description of change

We need to send the model name in the metric batches.

## QA steps

1. bootstrap 
2. add a model
3. deploy metered application
4. set SLA
5. make sure metrics are sent with the model's name

one relevant changes are made:
6. get user statement, make sure model name is filled in

## Documentation changes

None

## Bug reference

N/A

### CODE REVIEW

File: apiserver/facades/agent/metricsender/metricsender.go [hunk 1]
```diff
@@ -103,7 +104,8 @@ func SendMetrics(st ModelBackend, sender MetricSender, clock clock.Clock, batchS
 				heldBatches = append(heldBatches, m.UUID())
 				heldBatchUnits[m.Unit()] = true
 			} else {
-				wireData = append(wireData, ToWire(m))
+
```
Comment (cmars) [inline apiserver/facades/agent/metricsender/metricsender.go]: dd

**Comment by cmars** [comment]
!!build!!

File: apiserver/facades/agent/metricsender/metricsender_test.go [hunk 2]
```diff
@@ -82,6 +84,39 @@ func (s *MetricSenderSuite) TestToWire(c *gc.C) {
 	c.Assert(result, gc.DeepEquals, expected)
 }
 
+func (s *MetricSenderSuite) TestSendMetricsFromNewModel(c *gc.C) {
```
[the remark this answers is not in the document]
Comment (alesstimec) [reply apiserver/facades/agent/metricsender/metricsender_test.go]: existing tests report model name as "controller".. which is not what i wanted to check..

[the remark this answers is not in the document]
Comment (cmars) [reply apiserver/facades/agent/metricsender/metricsender_test.go]: @alesstimec That seems good reason to have a separate test; we don't want to see all the metrics start showing up as coming from the controller model.

**Comment by alesstimec** [comment]
$$merge$$

**Pull request merged**